src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1955,6 +1955,61 @@ public function get_test_rest_availability() {
 		return $result;
 	}
 
+	/**
+	 * Test if 'file_uploads' directive in PHP.ini is turned off
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array The test results.
+	 */
+	public function get_test_file_uploads() {
+		$result = array(
+			'label'       => __( 'File uploads is on in PHP ini' ),
+			'status'      => 'good',
+			'badge'       => array(
+				'label' => __( 'Media' ),
```

Comment:
We've limited the core labels to `Security` and `Performance` to not overload it, this feels `Performance` is the most appropriate one (especially as uploads may not always be related to media files)